diff --git a/node_modules/react-native/jest/setup.js b/node_modules/react-native/jest/setup.js
index 3738bd2..eb9efef 100644
--- a/node_modules/react-native/jest/setup.js
+++ b/node_modules/react-native/jest/setup.js
@@ -17,12 +17,19 @@ jest.requireActual('@react-native/polyfills/error-guard');
 
 global.__DEV__ = true;
 
-global.performance = {
-  now: jest.fn(Date.now),
-};
+// Ref: https://github.com/facebook/react-native/issues/36440
+if (!global.performance) {
+  global.performance = {
+    now: jest.fn(Date.now),
+  };
+}
 
 global.regeneratorRuntime = jest.requireActual('regenerator-runtime/runtime');
-global.window = global;
+
+// Ref: https://github.com/facebook/react-native/issues/36440
+if (!global.window) {
+  global.window = global;
+}
 
 global.requestAnimationFrame = function (callback) {
   return setTimeout(callback, 0);
